### ISSUE TITLE
ArrayIndexOutOfBoundsException: -1

### ISSUE THREAD

**Issue by Rsslone** [malte0811/IndustrialWires#36]
I have a player that said he was building an ImmersiveEngineering diesel generator and when he went to form the multiblock his client crashed. His words of what exactly he did and the reports below.

"I took the hammer from immersive engineering and right clicked on the un created diesel generator blocks to try and form the multiblock. It instantly crashed my client and anyone that goes within render distance of my base crashes with the same error."

Client Crash Log - https://paste.dimdev.org/nosikufofa.mccrash
Client Latest.Log - https://paste.dimdev.org/wakefanano.coffeescript

An interesting bit is that even though he said he was only working with Immersive stuff, there is an IndustrialWires block somewhere. I asked him if he was using anything from Industrial Wires and he said no. I don't know how reliable that info is but here is the block information I got from WebAPI.

`[[[{"data":{"facing":"NORTH","type":"END"},"type":{"id":"industrialwires:mech_mb","name":"tile.industrialwires.mech_mb.name"}}]]]`

**Comment by Rsslone** [comment]
I managed to use world edit to remove the offending block. Here is what the thing looks like. I broke away some of the blocks that are freaking out and there is this ghost block in the middle. 

https://i.imgur.com/6YYLhaf.png

**Comment by malte0811** [comment]
Duplicate of #34. A build with a fix for this will be on CF once it is approved.
